src/otx/recipe/classification/multi_class_cls/otx_mobilenet_v3_large.yaml

```diff
@@ -12,9 +12,6 @@ optimizer:
     weight_decay: 0.0001
 
 scheduler:
-  - class_path: otx.algo.schedulers.warmup_schedulers.LinearWarmupScheduler
-    init_args:
-      num_warmup_steps: 10
```

Comment:
Why is this model missing the `warmup_steps` setting?

Reply:
Done, https://github.com/openvinotoolkit/training_extensions/pull/3035/commits/3ccd4b1946381915bc2c179da8746a23a82f9a56